cardano-explorer/src/Explorer/Web/ClientTypes.hs

```diff
@@ -0,0 +1,264 @@
+{-# OPTIONS_GHC -fno-warn-unused-top-binds #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings   #-}
+
+-- | Types that arise in the API: mostly simplified representations
+-- of the core types which are easier to serialize.
+-- Used in purescript-bridge.
+
+module Explorer.Web.ClientTypes
+       ( CHash (..)
+       , CAddress (..)
+       , CTxId (..)
+       , CBlockEntry (..)
+       , CTxEntry (..)
+       , CBlockSummary (..)
+       , CAddressType (..)
+       , CAddressSummary (..)
+       , CTxBrief (..)
+       , CUtxo  (..)
+       , CNetworkAddress (..)
+       , CTxSummary (..)
+       , CBlockRange (..)
+       , CGenesisSummary (..)
+       , CGenesisAddressInfo (..)
+       , CAddressesFilter (..)
+       , CCoin
+       , CAda (..)
+       , CByteString (..)
+       , toCHash
+       ) where
+
+import qualified Prelude
+import           Universum
```

Comment:
Can we drop `Universum` ? We should either use the standard prelude or `cardano-prelude`/